### PR TITLE
[CU-86bx26564] Collage section

### PR DESCRIPTION
**Pull request by CatalinGheorghiu**

### CODE REVIEW

**Comment by CatalinGheorghiu** [comment]
Task linked: [CU-86bx26564 Collage](https://app.clickup.com/t/86bx26564)

File: src/components/sections/collage/CollageCard.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import ProductDetails from "@/components/sections/collage/ProductDetails";
+import CollectionDetails from "@/components/sections/collage/CollectionDetails";
+
+type CollageCardProps = { item: CollageItemType };
+
+const CollageCard = ({ item }: CollageCardProps) => {
+  return (
+    <div
+      className={cn(
+        "",
```
Comment (ivanms1) [inline src/components/sections/collage/CollageCard.tsx]: remove empty class please :pray:

File: src/components/sections/collage/Collage.tsx [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+  images: string[];
+  type: "collection";
+  featured: boolean;
+};
+
+export type CollageItemType =
+  | CollectionType
+  | (ProductType & { type: "product"; featured: boolean });
+
+const collageItems: CollageItemType[] = [
```
Comment (ivanms1) [inline src/components/sections/collage/Collage.tsx:36]: maybe move this to a const file or at the bottom of this file under the component?
so you don't have to scroll too much to get to the component.

**Review by ivanms1 — APPROVED** [review]
🚀

**Pull request merged**